src/components/DelegationDashboard/DelegateCard.tsx

```diff
@@ -115,136 +158,273 @@ const DelegateCard = ({ delegate, className, trackNum, disabled }: Props) => {
 					)}
 					{!!delegate?.dataSource?.includes(EDelegateSource.NOVA) && (
 						<div className='flex items-center space-x-1 '>
-							<ImageIcon
-								src='/assets/delegation-tracks/nova-wallet.svg'
-								alt='nova wallet icon'
+							<Image
+								src={'/assets/delegation-tracks/nova-wallet.svg'}
+								height={20}
+								width={20}
+								alt=''
+								className={'sm:hidden'}
+							/>
+							<Image
+								src={'/assets/delegation-tracks/nova-wallet.svg'}
+								height={22}
+								width={22}
+								alt=''
+								className={'hidden sm:block'}
 							/>
-							<span className='text-xs font-normal text-bodyBlue dark:text-blue-dark-high'>Nova Wallet</span>
+							<span className={`${poppins.variable} ${poppins.className} text-[10px] font-normal text-bodyBlue dark:text-blue-dark-high sm:text-xs`}>Nova Wallet</span>
 						</div>
 					)}
 				</div>
 			) : (
 				<>
 					{!!delegate?.dataSource?.includes(EDelegateSource.W3F) && (
-						<div className='ml-[-0.6px] mr-[-0.6px] mt-[-1px] flex h-8 items-center gap-1 rounded-t-md bg-[#272525] px-5'>
-							<W3FIcon />
-							<span className='text-xs font-normal text-white dark:text-blue-dark-high '>W3F Delegate</span>
+						<div className='ml-[-0.6px] mr-[-0.6px] mt-[-1px] flex h-8 items-center gap-1 rounded-t-md bg-[#272525] px-3 sm:px-5'>
+							<Image
```

Comment:
just use isMobile keyword to check display instead of hidden classaname and two times initialization of <Image